tools/Gemfile.plugins.all

```diff
@@ -1,6 +1,5 @@
 require 'octokit'
-skiplist = Regexp.union([ /^logstash-codec-cef$/, /^logstash-input-gemfire$/, /^logstash-output-gemfire$/,
-    /^logstash-input-couchdb_changes$/, /^logstash-filter-metricize$/, /^logstash-filter-yaml$/, /example/])
+skiplist = Regexp.union([ /^logstash-codec-cef$/, /^logstash-input-gemfire$/, /^logstash-output-gemfire$/, /^logstash-filter-metricize$/, /^logstash-filter-yaml$/, /jms$/, /example$/])
```

Comment:
nope, previously couchdb was not published, but we recently did. also, couch is now shipped by default